### PR TITLE
Replace Bridge::FILES by Content::FILES

### PR DESCRIPTION
**Pull request by yguedidi**

### CODE REVIEW

File: packages/Parallel/ValueObject/Bridge.php [hunk 1]
```diff
@@ -24,11 +24,6 @@ final class Bridge
      */
     public const SYSTEM_ERRORS_COUNT = 'system_errors_count';
 
-    /**
-     * @var string
-     */
-    public const FILES = 'files';
```
Comment (yguedidi) [inline packages/Parallel/ValueObject/Bridge.php:30]: is this a BC break? I don't think so as this enum looks internal, and no `@api` annotation

**Comment by yguedidi** [comment]
@TomasVotruba @samsonasik anything else needed for this? :)

**Comment by TomasVotruba** [comment]
This would break BRIDGE consistency, as all other variables are used. The Easy Parallel should be used only for its internal code.

Personally, I'd prefer PRs that improve something for the user, rather than detailed constant removals.
It feel better to merge those as they will be part of release notes :)

**Comment by yguedidi** [comment]
@TomasVotruba my guess here is that the usage of the Bridge::FILES is wrong, maybe a forgotten migration.
so this PR align to use the right key

see https://github.com/rectorphp/rector-src/blob/main/packages/Parallel/Application/ParallelFileProcessor.php#L99

**Comment by yguedidi** [comment]
@TomasVotruba don't you think it's good to align the constant used when writing with the one used when reading?

**Comment by yguedidi** [comment]
@TomasVotruba should I reopen this PR based on the synchronization argument? seems that you like things to be synchronized even with other projects :)

**Pull request closed without merging**